setup.cfg

```diff
@@ -1,4 +1,4 @@
-[build_sphinx]
+[build_docs]
```

Comment:
I would like to keep both if possible, being one of the mummies who does things the old way...

Reply:
OK done